FastSimulation/Tracking/python/GlobalPixelSeedProducerForElectrons_cff.py

```diff
@@ -44,9 +43,7 @@
     'FPix1_pos+FPix2_pos',
     'FPix1_neg+FPix2_neg',
 )
-globalPixelSeedsForPhotons.skipSimTrackIdTags = [cms.InputTag("globalPixelStepIds")]
-globalPixelSeedsForPhotons.outputSeedCollectionName = 'GlobalPixel'
-globalPixelSeedsForPhotons.zVertexConstraint = cms.double(-1)
+globalPixelSeedsForPhotons.simTrackSelection.skipSimTrackIds = [cms.InputTag("globalPixelStepIds")]
 globalPixelSeedsForPhotons.originRadius = cms.double(0.02)
 globalPixelSeedsForPhotons.originpTMin = cms.double(1.5)
```

Comment:
@matt-komm @shchauha

how did we arrive at the values for originRadius and originpTMin?
why is the zVertexConstraint removed?